add JuliaCon22 blogpost (#72)
JuliaSmoothOptimizers/JuliaSmoothOptimizers.github.io: news-and-blogposts/2022/2022-07-30-juliacon22.md

@def title = "Conference: JuliaSmoothOptimizers at JuliaCon 2022"

> @def rss_description = "This year JuliaCon 2022/ JuMP-dev 2022 featured three talks on JuliaSmoothOptimizers."

Reviewer: ```suggestion
@def rss_description = "This year's JuliaCon 2022/ JuMP-dev 2022 featured three talks on JuliaSmoothOptimizers."
```

# Conference: JuliaCon 2022

> This year [JuliaCon 2022](https://juliacon.org/2022)/ JuMP-dev 2022 featured three talks on JuliaSmoothOptimizers during the [JuMP-dev](https://jump.dev/meetings/juliacon2022/) stream.

Reviewer 2: ```suggestion
This year's [JuliaCon 2022/JuMP-dev 2022](https://juliacon.org/2022) featured three talks on JuliaSmoothOptimizers during the [JuMP-dev](https://jump.dev/meetings/juliacon2022/) stream.
```

> This year's [JuliaCon 2022/JuMP-dev 2022](https://juliacon.org/2022) featured three talks on JuliaSmoothOptimizers during the [JuMP-dev](https://jump.dev/meetings/juliacon2022/) stream.

Reviewer: ```suggestion
This year's [JuliaCon 2022/JuMP-dev 2022](https://juliacon.org/2022) featured three talks on JuliaSmoothOptimizers in the [JuMP-dev](https://jump.dev/meetings/juliacon2022/) stream.
```

> Dominique Orban gave a complete overview in 20 minutes (!!) of the JuliaSmoothOptimizers organization. The talk mentions all the key packages to model optimization problem ([NLPModels.jl](https://github.com/JuliaSmoothOptimizers/NLPModels.jl) & co), solve the problem ([JSOSolvers.jl](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl) & co) and do simple yet amazing benchmarks with [SolverBenchmark.jl](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl).

Reviewer: 25 minutes?

> Dominique Orban gave a complete overview in 20 minutes (!!) of the JuliaSmoothOptimizers organization. The talk mentions all the key packages to model optimization problem ([NLPModels.jl](https://github.com/JuliaSmoothOptimizers/NLPModels.jl) & co), solve the problem ([JSOSolvers.jl](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl) & co) and do simple yet amazing benchmarks with [SolverBenchmark.jl](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl).

Reviewer: ```suggestion
Dominique Orban gave a complete overview in 20 minutes (!!) of the JuliaSmoothOptimizers organization. The talk mentions all the key packages to model optimization problems ([NLPModels.jl](https://github.com/JuliaSmoothOptimizers/NLPModels.jl) & Co), solve models ([JSOSolvers.jl](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl) & Co), and do simple yet amazing benchmarks with [SolverBenchmark.jl](https://github.com/JuliaSmoothOptimizers/JSOSolvers.jl).
```

> Geoffroy Leconte presented [RipQP.jl](https://github.com/JuliaSmoothOptimizers/RipQP.jl) a multi-precision algorithm for convex quadratic optimization. The multi-precision feature illustrates the strength of pure Julia implementations, and the performance is there.

Reviewer: ```suggestion
Geoffroy Leconte presented [RipQP.jl](https://github.com/JuliaSmoothOptimizers/RipQP.jl), a multi-precision algorithm for convex quadratic optimization. The multi-precision feature illustrates the strength of pure Julia implementations, and the ensuing performance.
```

> Tangi Migot showed an application of the JuliaSmoothOptimizers framework for PDE-constrained optimization modeled by [PDENLPModels.jl](https://github.com/JuliaSmoothOptimizers/PDENLPModels.jl).

Reviewer: ```suggestion
Tangi Migot showed an application of the JuliaSmoothOptimizers framework to PDE-constrained optimization problems modeled with [PDENLPModels.jl](https://github.com/JuliaSmoothOptimizers/PDENLPModels.jl).
```
